.gitignore

```diff
@@ -29,3 +29,4 @@ coverage.xml
 docs/_build/
 docs/_templates/
 cover
+.idea/
```

Comment:
if you could actually just add this to your git exclude file, that'd be great (since it's specific to *your* set up).  e.g. within the main `ramlfications` directory, `$ echo ".idea" >> .git/exclude`